.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
There is now a [`docker.yml` workflow](https://github.com/NodeBB/NodeBB/blob/master/.github/workflows/docker.yml) - so I think this can be just deleted?